examples/jsm/renderers/common/Bindings.js

```diff
@@ -124,20 +124,26 @@ class Bindings extends DataMap {
 
 			} else if ( binding.isSampler ) {
 
-				binding.update();
+				const updated = binding.update();
```

Comment:
I think `NodeSampler.update()` should return `undefined`?

Reply:
Oh Indeed! I was confused with the Sampler/Sampled name but it appears that the issue was only related to SampledTexture. I did revert my changes regarding Sampler!